src/Core/Content/ImportExport/Strategy/Import/OneByOneImportStrategy.php

```diff
@@ -49,6 +49,7 @@ public function import(
                 $result = $this->repository->upsert([$record], $context);
             }
 
+            // @deprecated tag:v6.7.0 - remove this event with no replacement
```

Comment:
see above. Another benefit: extensions can already test with the major flag if they still work